test/monitoring/monitoring.go

```diff
@@ -45,7 +46,7 @@ func CheckPortAvailability(port int) error {
 // GetPods retrieves the current existing podlist for the app in monitoring namespace
 // This uses app=<app> as labelselector for selecting pods
 func GetPods(kubeClientset *kubernetes.Clientset, app, namespace string) (*v1.PodList, error) {
-	pods, err := kubeClientset.CoreV1().Pods(namespace).List(metav1.ListOptions{LabelSelector: fmt.Sprintf("app=%s", app)})
+	pods, err := kubeClientset.CoreV1().Pods(namespace).List(context.Background(), metav1.ListOptions{LabelSelector: fmt.Sprintf("app=%s", app)})
```

Comment:
should this be plumbed like the other test helpers?

Reply:
done in 77678bb83f220dd422bf886e7ce009f6f23d6616